### DISCUSSION TITLE
Installing Deepmd-Kit + Lammps with Cuda Version 11.1

### DISCUSSION THREAD

**Discussion by AvMach** [deepmodeling/deepmd-kit#404] [Q&A]
Hello,

As I upgraded to a RTX 3080 recently and want to use it for calculations in Lammps with Deepmd and it only supports Cuda 11.1, I want to ask how one should proceed when trying to install it. Is the latest v1.3.2 installer enough as the improvement mentioned: "Compiling with Cuda 11.0 and 11.1 #390" or do I have to do a manual installation?

I tried installing it with the run file but it seems that there is a problem with the installation as an error with Cuda 10.x is specified.

Thank you, 

Alex

**Comment by njzjz** [comment]
For CUDA>=11, publishing a binary version is in our plan. However, currently you have to compile by yourself. I suggest to compile TensorFlow 2.4.1 following [the document](https://github.com/deepmodeling/deepmd-kit/blob/7e534be15448fafbca82b6b71b540dde81edb373/doc/install-tf.2.3.md) (this document is for v2.3, but v2.4 should be OK).
